projects/sandbox/infer/infer/deploy.py

```diff
@@ -134,7 +134,8 @@ def main(
 
     num_shifts = get_num_shifts(data_dir, Tb, shift)
     parameters = "shift0,shift1,seq_id\n"
-    for i in range(num_shifts):
+    # skip the 0lag shift
+    for i in range(1, num_shifts + 1):
```

Comment:
Done. Can you explain why the min `sequence_id `matters? I thought it was just some identifier for the segment that just has to be unique?

Reply:
It doesn't necessarily for what we're doing now, but you can imagine more exotic scenarios where one or multiple users are doing inference runs pointing at the same server, and it's important to ensure that they know that they'll get the sequence IDs they asked for so that they know they won't step on each other's toes.